src/mumble/Themes.cpp

```diff
@@ -55,25 +63,59 @@ void Themes::applyFallback() {
 }
 
 bool Themes::applyConfigured() {
+	static QString currentThemePath;
+
+
 	boost::optional< ThemeInfo::StyleInfo > style = Themes::getConfiguredStyle(Global::get().s);
 	if (!style) {
 		return false;
 	}
 
-	const QFileInfo qssFile(style->getPlatformQss());
+	QString themePath;
+	if (style->themeName == "Auto" || style->name == "Auto") {
+#if QT_VERSION >= QT_VERSION_CHECK(6, 5, 0)
+		auto colorScheme = QGuiApplication::styleHints()->colorScheme();
```

Comment:
We usually do not want to use ``auto`` in the Mumble source code (exceptions apply :laughing: ), but you should convert this to a bool directly anyway, as mentioned above.